packages/post-purchase-ui-extensions/src/components/Layout/README.md

```diff
@@ -10,7 +10,7 @@ optional = ?
 | inlineAlignment? | <code>"leading" &#124; "trailing"</code> | Specifies the inline alignment of the layout in its container. By default, it will be centered.  |
 | blockAlignment? | <code>"center" &#124; "trailing"</code> | Specifies the block alignment of the layout in its container. By default, it will be leading.  |
 | maxInlineSize? | <code>number</code> | Default maximum inline size of the layout within its viewport. The size specified will constrain the space available for its sections and will be centered in the viewport unless specified otherwise with `inlineAlignment`.<br /><br />Numbers less than or equal to 1 are treated as percentages and numbers greater than 1 are treated as pixels.<br /><br />Examples: - `500` represents `500px` - `0.5` represents `50%` - `1` represents `100%`  |
-| sizes? | <code>"auto" &#124; "fill" &#124; number[]</code> | Default sizes for each section of the layout  |
+| sizes? | <code>("auto" &#124; "fill" &#124; number)[]</code> | Default sizes for each section of the layout  |
```

Comment:
Unrelated change caused by a recent PR that now wraps arrays inside parentheses.